en/components/autocomplete.md

```diff
@@ -0,0 +1,254 @@
+</p>
+<div class="divider"></div>
+
+### Demo
+The following sample demonstrates simple use of the `igx-autocomplete` directive, applied on an `igx-input`. The directive is provided with an `igx-drop-down` which has a defined filtering pipe.
+<div class="sample-container loading" style="height: 400px;">
+    <iframe id="autocomplete-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/data-entries/autocomplete" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="autocomplete-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+> [!WARNING]
+> To start using Ignite UI for Angular components in your own projects, make sure you have configured all necessary dependencies and have performed the proper setup of your project. You can learn how to do this in the [**installation**](https://www.infragistics.com/products/ignite-ui-angular/getting-started#installation) topic.
+
+## Usage
+### Getting Started
+To use the [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) we need to import the **IgxAutocompleteModule** and **IgxDropDownModule** in our **app.module**. If the [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) is applied on an [igx-input]({environment:angularApiUrl}/classes/igxinputdirective.html), the **igxInputGroupModule** is also required:
+
+```typescript
+// app.module.ts
+
+...
+import { 
+    IgxAutocompleteModule,
+    IgxDropDownModule, 
+    IgxInputGroupModule 
+} from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [
+        ..., 
+        IgxAutocompleteModule,
+        IgxDropDownModule,
+        IgxInputGroupModule,
+        ....
+    ],
+    ...
+})
+export class AppModule {}
+```
+
+Then adding the `igxAutocomplete` directive with a value, which is reference to a drop down:
+
+```html
+<igx-input-group>
+    <input igxInput name="towns" type="text"
+        [igxAutocomplete]='townsPanel'/>
+    <label igxLabel for="towns">Towns</label>
+</igx-input-group>
+<igx-drop-down #townsPanel>
+    <igx-drop-down-item *ngFor="let town of towns">
+        {{town}}
+    </igx-drop-down-item>
+</igx-drop-down>
+```
+
+```typescript
+@Component({
+    selector: 'app-autocomplete-sample',
+    styleUrls: ['autocomplete.sample.css'],
+    templateUrl: `autocomplete.sample.html`
+})
+export class AutocompleteSampleComponent {
+    constructor() {
+        this.towns = [ 'Sofia', 'Plovdiv', 'Varna', 'Burgas'];
```

Comment:
Let's make those at least 10 - so the user can have the feeling for the filtering if the code snippet is used